ci-operator/step-registry/openshift/OWNERS

```diff
@@ -9,3 +9,5 @@ approvers:
 - ewolinetz
 - csrwng
 - staebler
+- periklis
+- blockloop
```

Comment:
This isn't Loki-scoped either.

Reply:
odd, i don't recall adding them here specifically... could this be a result of `make registry-metadata` ?